doc/stanzas/ignored_subdirs.rst

```diff
@@ -1,7 +1,9 @@
 .. _dune-ignored_subdirs:
 
-ignored_subdirs (Deprecated in 1.6)
------------------------------------
+ignored_subdirs
```

Comment:
Maybe time to grt rid of this completely?

Reply:
I'd prefer keeping the content PRs separate from the metadata PRs, but in this particular case `ignored_subdirs` is still supported (though with a warning).